File: c/exo_13.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>


int main(void)
{
  int i, x;
  char numb[5] = {0};
  printf("Combien d'email à générer ?\n> ");
  fgets(numb, 5, stdin);
  numb[strcspn(numb, "\n")] = 0;
  if(atoi(numb) > 50){
      printf("Ça fait beaucoup trop !\n");
      exit(-1);
  } else
        int list[atoi(numb)];
  int y = date - atoi(numb);
  for (i = atoi(numb); i <= date; ++i){
	  printf("Il y a %d ans, tu avais %d ans\n", y, x);
          x = x + 1;
          y = y - 1;
  }
}

File: rb/exo_13.rb
# frozen_string_literal: true

print("Combien d'email à générer ?\n> ")
numb = gets.chomp
i = 0

abort('Ça fait beaucoup trop !') if numb.to_i > 50

Array
  .new(numb.to_i) { |i| "jean.dupont.#{(i + 1).to_s.rjust(2, '0')}@email.fr" }
  .each { |email| puts email }
